bin/v-add-web-domain-backend

```diff
@@ -41,11 +41,12 @@ check_hestia_demo_mode
 
 # Defining pool directory
 prepare_web_backend
+get_domain_values 'web'
 
 # Checking backend configuration
-if [ -e "$pool/$backend_type.conf" ]; then
-    exit
-fi
+#if [ -e "$pool/$backend_type.conf" ]; then
+#    exit
+#fi
```

Comment:
Should this stay commented out or was it during debugging? If it isnt needed, we probaly should remove it.